### PR TITLE
added patch and delete methods for lists

### PR DESCRIPTION
**Pull request by SamDaltonJr**
tested and it's looking good

### CODE REVIEW

**Comment by SamDaltonJr** [comment]
Still need to merge/add error stuff

File: routes/lists.js [hunk 1]
```diff
@@ -11,23 +11,76 @@ const jwtAuth = passport.authenticate('jwt', { session: false });
...
+router
+  .route('/:id')
+  .get((req, res, next) => {
+    const { id: userId } = req.user;
+    const { id } = req.params;
+    //TODO: check that id is an ObjectId
+    User.findById(userId)
+      .then(user => {
+        const list = user.shoppingLists.id(id);
+        console.log(list);
```
Comment (mischb) [inline routes/lists.js]: remove console log

**Comment by ender554** [comment]
running well locally

File: routes/lists.js (hunk 1 above)
Comment (SamDaltonJr) [reply routes/lists.js]: done

**Pull request merged**